### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by NguyenAiSi** [JayChiapero/JayChiapero.github.io#1]
Hello

Is it possible to get the .deb of Layers ? 
I don't care about what people say so if you can share it 
Thanks for your work

**Comment by JayChiapero** [comment]
Yes, I will release the .deb for Layers to anyone who asks. The only way I know to give it to people is emailing them the .deb. If you’re okay with that, it’s all yours. Eventually, I’ll get it updated to remove bugs for it. I’m sorry for all the accusations towards me. I didn’t take StackXI’s code. I simply saw it was open source and knowing it had a lot of bugs, I figured I could use the code as a template to create my own tweak with more features, less bugs and a smoother user experience. I should’ve went to the dev for StackXI originally but in my own opinion, I believe if a tweak is uploaded as open source, if you change the code in anyway then it’s fair game. I changed a lot within the code, as well as writing my own stuff to it. I did not know exactly how I’d change the variables at the time of making the tweak as it is my first “big” tweak but my tweak was not even supposed to be released until I made it work properly for my liking, then I planned on adding other features suggested by you guys. I’m an upcoming developer in hopes of making a more positive impact on the jailbreak community, I’m sorry for all the negativity towards me, “stealing” code & making a bad name for myself. If anyone is interested in the .deb for Layers while I get it all situated and plan an update, please email me at tge.johnson@gmail.com. I will gladly give you the .deb.

**Comment by NguyenAiSi** [comment]
Thanks for your quick answer
I just send you a email on tge.johnson@gmail.com

Thanks
